### PR TITLE
Moar leds

### PR DESCRIPTION
**Pull request by ethanprivette**

### CODE REVIEW

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -238,7 +238,7 @@ private rgbValues(int r, int g, int b) {
   public static String VISION_LIMELIGHT = "limelight-vanap";
   public static String DRIVER_LIMELIGHT = "limelight-panav";
 
-    public static final Mode currentMode = Mode.REAL;
+    public static final Mode currentMode = Mode.SIM;
```
Comment (MGuron) [inline src/main/java/frc/robot/Constants.java]: danger

File: src/main/java/frc/robot/subsystems/Leds.java [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+        wave(allianceColor, secondaryDisabledColor, waveAllianceCycleLength, waveAllianceDuration);
+      }
+    }
+      if (DriverStation.isAutonomous()) {
+        wave(Color.kDarkGreen, Color.kPurple, waveSlowCycleLength, waveSlowDuration);
+        if (autoFinished) {
+          double fullTime = (double) length / waveFastCycleLength * waveFastDuration;
+          solid((Timer.getFPGATimestamp() - autoFinishedTime) / fullTime, Color.kGreen);
+        }
+      } else { // Enabled
```
Comment (MGuron) [inline src/main/java/frc/robot/subsystems/Leds.java:162]: The existing colors for intaking should be maintained to reduce driver confusion

File: src/main/java/frc/robot/commands/ShootCommand.java [hunk 3]
```diff
@@ -49,13 +45,13 @@ public void execute() {
     if(shooter.flywheelUpToSpeed(shooterPositions.getShootSpeed() * 0.875)){
         indexer.setIndexerSpeed(Constants.INDEXER_FEED_SPEED);
         intake.setIntakeSpeed(0.5);
+        shooter.launchCommand().withTimeout(0.5).schedule();
```
Comment (MGuron) [inline src/main/java/frc/robot/commands/ShootCommand.java:50]: LaunchCommand causes a runtime error with real target, add logic to only run launchCommand in Sim mode

File: src/main/java/frc/robot/subsystems/Leds.java (hunk 2 above)
Comment (ethanprivette) [reply src/main/java/frc/robot/subsystems/Leds.java:162]: panav said these are fine plus this is what he wanted originally so

**Review by SpeedyCoder37 — APPROVED** [review]
LEDs 😍

**Pull request merged**